content/en/docs/contribute/style/style-guide.md

```diff
@@ -38,29 +38,25 @@ The English-language documentation uses U.S. English spelling and grammar.
 
 {{< comment >}}[If you're localizing this page, you can omit the point about US English.]{{< /comment >}}
 
-## Documentation content standards
+## What Kubernetes documentation does and doesn't contain
```

Comment:
hello @aimeeu . Thanks for working on the style guide.
nit: Could the title be shortened to something like: `Guidelines for contributing content`

Reply:
Updated - I like your suggestion